frontend/document/document.lua

```diff
@@ -5,10 +5,10 @@ local Configurable = require("configurable")
 local DrawContext = require("ffi/drawcontext")
 local Geom = require("ui/geometry")
 local Math = require("optmath")
-local Screen = require("device").screen
 local TileCacheItem = require("document/tilecacheitem")
 local lfs = require("libs/libkoreader-lfs")
 local logger = require("logger")
+local Runtimectl = require("runtimectl")
```

Comment:
Per the order it looks like an uppercase class.
https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires